ssp

```diff
@@ -8252,13 +8240,36 @@ sub check_for_cgls {
 ## END malware checks
 
 sub build_rpm_list {
-    return if @extended_rpm_list;    # already did this, don't do it again
+    my ($rpm_list) = @_;
+    return unless $rpm_list;
+    return map {
+        my ( $arch, $name, $version, $release ) = split( /\t/, $_ );
+        {
+            'name'    => defined $name    ? $name    : '',
+            'arch'    => defined $arch    ? $arch    : '',
+            'version' => defined $version ? $version : '',
+            'release' => defined $release ? $release : '',
```

Comment:
Small and stylistic but I'd like to see any declarations alphabetized within their block.  nit-picky so I am not going to ding here.